librocksdb_sys/crocksdb/c.cc

```diff
@@ -2595,6 +2625,25 @@ void crocksdb_options_set_vector_memtable_factory(crocksdb_options_t* opt, uint6
   opt->rep.memtable_factory.reset(new VectorRepFactory(reserved_bytes));
 }
 
+void crocksdb_load_latest_options(const char* dbpath, crocksdb_env_t* env,
```

Comment:
How can we know that if the options file does not exist or there is some error related to it?
Maybe you can return false when the options file is not found?